src/app/lib/layoutGrid.js

```diff
@@ -10,6 +10,14 @@ import {
   group5ScreenWidthMin,
 } from './constants/styles';
 
+export const twelveEightyGridWrapper = css`
+  @media (min-width: ${group5ScreenWidthMin}) {
+    display: grid;
```

Comment:
Been thinking about this and I'm not convinced this needs to be `display: grid`. All we need is a `div` with max-width set. I also am not convinced we will have a use case for this in css other than in main content so I'm currently thinking this could actually be a styled div defined within the MainContent component.